packages/block-library/src/latest-comments/index.php

```diff
@@ -112,9 +119,15 @@ function render_block_core_latest_comments( $attributes ) {
 				);
 			}
 			$list_items_markup .= '</footer>';
-			if ( $attributes['displayExcerpt'] ) {
+			
+			// Handle different display content options
+			if ( 'full' === $display_content ) {
+				$list_items_markup .= '<div class="wp-block-latest-comments__comment-excerpt">' . wpautop( get_comment_text( $comment ) ) . '</div>';
+			} elseif ( 'excerpt' === $display_content ) {
 				$list_items_markup .= '<div class="wp-block-latest-comments__comment-excerpt">' . wpautop( get_comment_excerpt( $comment ) ) . '</div>';
 			}
+			// If 'none', don't add any content
+			
```

Comment:
```suggestion
			if ( 'full' === $display_content ) {
				$list_items_markup .= '<div class="wp-block-latest-comments__comment-excerpt">' . wpautop( get_comment_text( $comment ) ) . '</div>';
			} elseif ( 'excerpt' === $display_content ) {
$list_items_markup .= '<div class="wp-block-latest-comments__comment-excerpt">' . wpautop( get_comment_excerpt( $comment ) ) . '</div>';
}
```

- Remove unnecessary indents
- I think the comments are redundant